doc/rst/usingchapel/prereqs.rst

```diff
@@ -4,16 +4,11 @@
 Chapel Prerequisites
 ====================
 
-If you have a standard UNIX environment, a C/C++ compiler, some basic
-scripting languages, a GNU-compatible make, and awk installed you should
-have no problems getting started with Chapel.
+Chapel is designed to be portable to a variety of systems and depends on
```

Comment:
Really nitpicky (could easily be ignored), but I might say "only depends on" or maybe better "only requires" to try and emphasize the "this should be no big deal" message.